grader/src/test/java/edu/pdx/cs410J/grader/scoring/ProjectSubmissionOutputFileParserTest.java

```diff
@@ -234,9 +235,21 @@ public void parseTest1() {
 
 
   private ProjectSubmission parse(OutputFile file) {
+    ProjectSubmission projectSubmission = null;
+    try {
     String text = file.getText();
     ProjectSubmissionOutputFileParser parser = new ProjectSubmissionOutputFileParser(new StringReader(text));
-    return parser.parse();
+    projectSubmission= parser.parse();
+   
+       parser.parse();
+    } catch (ParseException e) {
```

Comment:
You might as well have the parse() method declare that it should throw the `ParseException`.